core/src/test/java/com/contentful/vault/ObserveTest.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2015 Contentful GmbH
+ * Copyright (C) 2018 Contentful GmbH
```

Comment:
shouldn't this be 2015 - 2018?

Reply:
Hmm, good question. Classically we always used current year for those headers.

Reply:
For consistency reasons, I'll just keep the header format how I have it everywhere else: One year, changed once a year ... :slightly_smiling_face: